Dockerfile

```diff
@@ -2,9 +2,9 @@
 # Use of this source code is governed by an Apache2
 # license that can be found in the LICENSE file.
 
-FROM gcr.io/distroless/base
+FROM gcr.io/distroless/static:nonroot
```

Comment:
Out of curiosity, is there a difference between the `static` and `base` image types? I looked around on the distroless readme and didn't spot any mention of the static ones.